content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -112,6 +113,59 @@ To initialize the control-plane node run:
 kubeadm init <args>
 ```
 
+### Considerations about network addresses
+
+There are several places where the network address(es) that various
+components advertise to use when contacting them appear, including the
+following.
+1. The Endpoints/EndpointSlice of the kube-apiserver.
+1. The Subject Alternative Name list of the kube-apiserver's X.509 certificate.
+1. The `kubeadm join` command printed by `kubeadm`.
+1. The `InternalIP` type of address of the Node.
+
+You can get your dessired address in all but the last place by using
```

Comment:
This statement is a little bit unclear to me. Am I "getting" addresses by using a command line flag?
Please help revise it so that non-native speakers can understand.